libraries/chain/include/eosio/chain/backing_store/db_combined.hpp

```diff
@@ -288,14 +327,34 @@ class rocksdb_contract_db_table_writer {
    }
 
    void check_context(const name& code, const name& scope, const name& table) {
-      if (table_context_ && table_context_->code == code &&
-          table_context_->scope == scope && table_context_->table == table) {
+      // this method only has to do with standalone processing
+      if (!is_standalone() || is_same_table(code, scope, table)) {
          return;
       }
-      table_context_ = table_id_object_view{code, scope, table, name{}, 0};
+      table_context_->code = code;
+      table_context_->scope = scope;
+      table_context_->table = table;
```

Comment:
This is for standalone context, it is just supplying code, scope, and table.

Reply:
? line 282 sets payer and total_count. line 279 sets payer.

Reply:
Not sure what to do with this comment.

Reply:
@brianjohnson5972 thoughts? My opinion is that it should be changed back.

Reply:
for standalone we don't have a valid count (ever) or payer (unless the standalone key is a table key).  We may need to discuss this in a call to get past this.

Reply:
I'm sorry, I missed what you were saying, that it should ensure payer gets set to name{} and count =0.  Sorry I was confusing this with the other and thought you wanted it set to the real value.  Yeah, it should be changed back, since if a table key was received earlier, then an erroneous payer name would be reported.

Reply:
just to confirm, these 3 lines should be changed back to:
`table_context_ = table_id_object_view{code, scope, table, name{}, 0};`
?

Reply:
yes

Reply:
Done.